packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -491,13 +491,17 @@ void _printPluginUpdatePubspecMessage(String pluginPath, String platformsString)
   globals.printStatus('''
 
 You need to update $pluginPath/pubspec.yaml to support $platformsString.
```

Comment:
If this feedback has been applied, I don't see it: https://github.com/flutter/flutter/pull/71962#discussion_r539713746